packages/vue/src/Mixin/Resource.js

```diff
@@ -21,7 +21,7 @@ export default {
   },
   methods: {
     loadResource() {
-      return this.$store.dispatch(`hyral_${this.resourceType}/LOAD_RESOURCE`, this.id);
+      return this.$store.dispatch(`hyral_${this.resourceType}/LOAD_RESOURCE`, this.id, (typeof this.parameterBag === 'undefined') ? {} : this.parameterBag);
```

Comment:
no typeof, you can use null coalesce:
https://stackoverflow.com/questions/476436/is-there-a-null-coalescing-operator-in-javascript

Reply:
In any case at least no {} instead of null. null = really empty, {} is not really empty and an actual value.